src/vs/editor/contrib/format/formatActions.ts

```diff
@@ -25,7 +25,7 @@ import { KeybindingWeight } from 'vs/platform/keybinding/common/keybindingsRegis
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 import { onUnexpectedError } from 'vs/base/common/errors';
 import { EditorOption } from 'vs/editor/common/config/editorOptions';
-import { Progress } from 'vs/platform/progress/common/progress';
+import { Progress, IProgressService, ProgressLocation } from 'vs/platform/progress/common/progress';
```

Comment:
Better use the `IEditorProgressService`. It renders progress in the context of an editor that's what we use for editor features, like rename, go-to, etc

Reply:
thanks, updated~